sematic/ui/packages/common/src/pages/RunSearch/SearchFilters.tsx

```diff
@@ -0,0 +1,29 @@
+import OrderSection from 'src/pages/RunSearch/filters/OrderSection';
+import OtherFiltersSection from 'src/pages/RunSearch/filters/OtherFilterSection';
+import OwnersFilterSection from 'src/pages/RunSearch/filters/OwnersFilterSection';
+import SearchTextSection from 'src/pages/RunSearch/filters/SearchTextSection';
+import StatusFilterSection from 'src/pages/RunSearch/filters/StatusFilterSection';
+import TagsFilterSection from 'src/pages/RunSearch/filters/TagsFilterSection';
+import Button from '@mui/material/Button';
+import styled from '@emotion/styled';
+import theme from 'src/theme/new';
+
+const StyledButton = styled(Button)`
+    margin: 0 -${theme.spacing(5)};
+    height: 50px;
+`;
+
+const SearchFilters = () => {
+    return <>
+        <SearchTextSection />
+        <TagsFilterSection />
+        <StatusFilterSection />
+        <OwnersFilterSection />
+        <OtherFiltersSection />
+        <OrderSection />
+        <StyledButton variant="contained" disableElevation>Filter runs</StyledButton>
+        <StyledButton variant="contained" disableElevation color={"white"}>Clear filters</StyledButton>
```

Comment:
Is it hard / wasteful to implement this functionality in this PR?

Reply:
This PR is pretty big now. If it's ok, I will do it in a separate PR. It will require implementing resetting in all of the filter sections, currently, there are only a couple implemented to demonstrate the possibility.